AlphaWallet/Tokens/Collectibles/Coordinators/TokensCardCollectionCoordinator.swift

```diff
@@ -117,7 +114,7 @@ class TokensCardCollectionCoordinator: NSObject, Coordinator {
     }
 
     private func transactionsFilter(for strategy: ActivitiesFilterStrategy, tokenObject: TokenObject) -> TransactionsFilterStrategy {
```

Comment:
Some func like this can be `functional`? Easier to spot that they aren't mutating (i.e. if they don't mutate the args). This particular one can be shared between `TokensCardCollectionCoordinator` and `SingleChainTokenCoordinator` too.